oracle/tests/conftest.py

```diff
@@ -141,13 +140,15 @@ def dd_environment():
 
 @pytest.fixture
 def bad_instance():
-    return {
-        "password": "badpassword",
-        "protocol": "TCP",
-        "server": "localhost:1521",
-        "service_name": "InfraDB.us.oracle.com",
-        "username": "datadog",
-    }
+    return deepcopy(
```

Comment:
I'm not sure I understand the need for `deepcopy` here. The fixture is function scoped, iiuc, which means it will recreate the dictionary for each test function that requests it. 

`deepcopy` to me only makes sense when we have some `INSTANCE` constant that we want to "recreate" in a function-scoped fixture.

Reply:
Not sure why I did that, we have to remove it from the other fixture. Thanks!